hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -1356,7 +1356,29 @@
       bus_host: none
       available_input_list: []
       available_output_list: []
-      available_inout_list: []
+      available_inout_list:
```

Comment:
are these intended to be temporary? i.e., once JTAG qualification is implemented in the pinmux, the goal would be to make these intermodule signals, right?

Reply:
this is something we need to talk to Arnon about.  The flash has a jtag interface that needs to be used during test mode. My assumption is that this can just be generic muxing.

Reply:
ok I see. that is actually the more convenient option, yes. the only thing we need to keep in mind is the muxing cost, since these are basically all-to-all muxes. the dedicated JTAG mux would only be present on certain pins on the other hand...

Reply:
yeah, so i also wanted to talk to Arnon how he expects to control the jtag. Because part of me thinks this should become another selection...